### ISSUE TITLE
crash when placing a call in airplaine mode

### ISSUE THREAD

**Issue by mcginty** [WhisperSystems/Signal-Android#4184]
not reproducible every time

```
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime: FATAL EXCEPTION: main
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime: Process: org.thoughtcrime.securesms, PID: 12552
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime: java.lang.IllegalStateException
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at android.media.MediaPlayer._start(Native Method)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at android.media.MediaPlayer.start(MediaPlayer.java:1219)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at org.thoughtcrime.redphone.audio.OutgoingRinger.onPrepared(OutgoingRinger.java:146)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at android.media.MediaPlayer$EventHandler.handleMessage(MediaPlayer.java:2593)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at android.os.Handler.dispatchMessage(Handler.java:102)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at android.os.Looper.loop(Looper.java:211)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at android.app.ActivityThread.main(ActivityThread.java:5373)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at java.lang.reflect.Method.invoke(Native Method)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at java.lang.reflect.Method.invoke(Method.java:372)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1020)
10-02 11:50:56.927 12552-12552/? E/AndroidRuntime:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:815)
10-02 11:50:56.934 731-1361/? D/audio_hw_primary: out_set_parameters: enter: usecase(1: low-latency-playback) kvpairs: routing=2
10-02 11:50:56.947 12552-14888/? E/MediaPlayer-JNI: QCMediaPlayer mediaplayer N
```

**Comment by moxie0** [comment]
ugh yeah gotta rewrite soa's old mediaplayer code

**Comment by cascheberg** [comment]
I did some possibly relevant rewriting in #4886

**Comment by Dyras** [comment]
https://gist.github.com/anonymous/07421f1f911f80fac5e3
I was just about to make a new issue about this, when I realized this issue happened again. This seems to be happening really often to me, are there any theories as to what causes this? I unlocked my phone when Signal was open when I locked it, and it crashed.

**Comment by agrajaghh** [comment]
fixed by #5357
